stack.yaml

```diff
@@ -1,74 +1,11 @@
-# This file was automatically generated by 'stack init'
```

Comment:
Could you please keep those default comments but only update sections that require changes?
(resolver and extra-deps)